package.json

```diff
@@ -7,7 +7,7 @@
   "repository": "https://github.com/oppiabot/oppiabot.git",
   "scripts": {
     "dev": "nodemon",
-    "lint": "eslint",
+    "lint": "eslint *.js *.json ./spec/**/*.json ./lib/*.js ./fixtures/*.json actions/**/*.js --fix",
```

Comment:
I think we should exclude JSON files.

Reply:
Sure sir.

Reply:
changes addressed in latest commit sir please review.